core/commands/storage.go

```diff
@@ -333,7 +338,17 @@ Use status command to check for completion:
 		if req.Options[testOnlyOptionName] != nil {
 			testFlag = req.Options[testOnlyOptionName].(bool)
 		}
-		go retryMonitor(context.Background(), api, ss, n, ssID, testFlag, isRepairMode, renterPid.Pretty())
+		p := 1
```

Comment:
can you put this number on line 139 with a `IntOption.(...).WithDefault(defaultPayoutPeriod)`?, then you don't need line 344-347